frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -348,13 +349,13 @@ def _stream():
 @backend_ns.route("/get-build-task/<task_id>/")
 @backend_ns.route("/get-build-task/<task_id>")
 def get_build_task(task_id):
-    try:
-        task = BuildsLogic.get_build_task(task_id)
-    except MalformedArgumentException:
```

Comment:
We are actually handling the same problem, and raising this exception inside `BuildsLogic.get_build_task`.  I'd prefer to handling the problem with just one method.  Perhaps we'd just change [first()](https://github.com/fedora-copr/copr/blob/5f30af8106c226823fb25eb447506b829c3d18d2/frontend/coprs_frontend/coprs/logic/builds_logic.py#L390) call to `one()`?  Then we should get the `NoResultFound` and it would be handled on line 357 here.

Reply:
Makes sense, I updated the PR, WDYT?
The change is in a separate commit for easier review.